### DISCUSSION TITLE
/colorpicker -> way to edit clicked color

### DISCUSSION THREAD

**Discussion by bobhenl** [Zrips/CMI#5065] [Feature Requests]
Hi, I wanna suggest when you use /colorpicker to get specific color, then it's printed into chat when you click on it. That's right, but the problem is, when you use for ex. Chat Control Pro that use different formatting for colors. Especially without (), so when I click on specific color in colorpicker, then send it as message, this happens:
![image](https://user-images.githubusercontent.com/18662086/105228748-3ab89f80-5b63-11eb-98cb-1739d56d1287.png)
![image](https://user-images.githubusercontent.com/18662086/105228761-3d1af980-5b63-11eb-8d6c-8ebb93f44e96.png)

So () aren't used in chatcontrolpro. And my feature request is that if we can change the format of it, so for ex. remove ()?

**Comment by CloudeLecaw** [comment]
normaly Chat Control Pro has to react here and change the behavior
the format CMI is putting it out is the official default hex code format
If ChatConrolPro does not support it they should add it. I suggest contacting the dev of this plugin about the fact and suggest the support there.

**Comment by bobhenl** [comment]
as you can see in chatcontrol is just #hex used, without (), so may Zrips can add way to remove (), but ok, will try to ask author of chc too

**Comment by Zrips** [comment]
@bobhenl 
It would be better for ChatControlPro to implement more universal format, as this allows us to have not only hex type colors like {#6600cc} or {#60c} or {#Purple} and utilize gradients with {#6600cc>}text{#purple<} Having plain #6600cc only works in specific situations and is restricted in its usage.

**Comment by CloudeLecaw** [comment]
u know request here got ignored by the devs mostly? This here is jsut for issues
send them here aggain: https://github.com/Zrips/CMI/discussions

**Comment by bobhenl** [comment]
I don't think so, when you look at discussion, I think that discussion got ignored... This section issues still resolve probably more feature requests than section discussion. If he should really put enhancements etc there, then it should be read more then now. Cause as I see Zrips reply more here probably.

**Comment by CloudeLecaw** [comment]
Maybe but heself was the one that said this. That requests and questtions here may got deleted because they made the discussion for this.
